src/test/resources/json/html.json

```diff
@@ -0,0 +1,71 @@
+[
```

Comment:
don't need to have separate JSON if you use current sample JSON or test via unit test